roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -61,16 +64,14 @@ class Interface(object):
             return True
 
         # veth interfaces named tap... belong to Neutron
-        if re.match(r'^tap[0-9a-z]{8}-[0-9a-z]{2}$', self.ifname) and \
-                self.iftype() == 'veth':
-            return True
+        if re.match(r'^tap[0-9a-z]{8}-[0-9a-z]{2}$', self.ifname):
+           if self.iftype() == 'tun' or self.iftype() == 'veth':
```

Comment:
We want to capture iftype=tun devices also so that we can plug these devices back to the bridges they belong. The check to skip resetting these devices are in destoy() now.